src/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.10)
+project(RawHash2Src LANGUAGES CXX)
+
+option(USE_CCACHE "Use ccache to speed up rebuilds" ON)
+option(USE_MOLD "Use mold linker for faster linking" ON)
+
+option(PYBINDING "Build with Python bindings" OFF)
+option(PROFILE "Enable profiling support" OFF)
+option(RUCLIENT_ENABLED "Enable ReadUntil client" OFF)
+
+option(ENABLE_ASAN "Enable AddressSanitizer" OFF)
+option(ENABLE_TSAN "Enable ThreadSanitizer" OFF)
+
+option(POD5_DOWNLOAD "Download POD5" ON)
+option(HDF5_COMPILE "Compile HDF5" ON)
+option(SLOW5_COMPILE "Compile slow5lib" ON)
+
+option(NOPOD5 "Do not use POD5" OFF)
+option(NOHDF5 "Do not use HDF5" OFF)
+option(NOSLOW5 "Do not use SLOW5" OFF)
```

Comment:
FYI: When this is ON, `rmap.cpp` does not compile because of [`__ac_Wang_hash`](https://github.com/CMU-SAFARI/RawHash/blob/ad5bf88e34b5c6d1f5c33bbae0ba2c6bd48ce534/src/rmap.cpp#L321), which seems to be part of Slow5.

Reply:
So we disable slow5 for now.
@canfirtina relevant for you to fix "multiple definitions of function" (error I think)

Reply:
It's actually "function is undefined" kind of error rather than "multiple definitions of functions". It seems that previously it was propagated from Slow5's include directory, even if the flag not to link it was set to true.